src/components/BankTransactions/BankTransactions.tsx

```diff
@@ -26,6 +26,17 @@ const COMPONENT_NAME = 'bank-transactions'
 const TEST_EMPTY_STATE = false
 const POLL_INTERVAL = 10000
 
+export type BankTransactionsMode = 'bookkeeping-client' | 'self-serve'
+
+export const categorizationEnabled: (
+  mode: BankTransactionsMode,
+) => boolean = mode => {
+  if (mode === 'bookkeeping-client') {
+    return false
+  }
+  return true
+}
```

Comment:
NIT: it's safe to ignore this suggestion: 

```suggestion
export const categorizationEnabled: (
  mode: BankTransactionsMode,
) => boolean = mode => mode !== 'bookkeeping-client'
```